src/Schema/PlasterManifest-v1.xsd

```diff
@@ -175,6 +175,11 @@
           <xs:documentation>The label for the choice that is presented in the user interface. The &amp; character must be respresented with the entity ref &amp;amp;</xs:documentation>
         </xs:annotation>
       </xs:attribute>
+      <xs:attribute name="condition" type="ptd:condition">
```

Comment:
Sorry, I didn't mean to drop off on this one. Thanks for tying this one off (and giving opportunities to allow me to complete it as well).